### PR TITLE
Add Screenshot#file_size_in_bytes method

### LINKED ISSUE #1: Add integer file size in bytes
To complement Rsteamshot::Screenshot#file_size, a new method #file_size_in_bytes would be nice. It should return an integer based on the #file_size string, parsed and converted to bytes.

### PR DESCRIPTION
**Pull request by marcqualie**
Fixes #1

This converts <String:"0.123 MB"> (#file_size) into <Integer:123000>

### CODE REVIEW

**Comment by marcqualie** [comment]
I put the unit multiplier as 1000 (rather than 1024). Wasn't sure which convention would be preferred/consistent with the rest of the library.

File: lib/rsteamshot/screenshot.rb [hunk 1]
```diff
@@ -92,6 +92,23 @@ def inspect
       self.class.name + '<' + JSON.generate(to_h) + '>'
     end
 
+    # Public: file_size parsed into bytes
+    #
+    # Returns a Integer
+    def file_size_in_bytes
+      unit_multiplier = 1000
+      size, unit = self.file_size.split(' ')
+      bytes = begin
```
Comment (cheshire137) [inline lib/rsteamshot/screenshot.rb]: I don't think the `begin`/`end` is necessary. You can just have `bytes = case unit.downcase` here.

File: spec/rsteamshot/screenshot_spec.rb [hunk 2]
```diff
@@ -82,4 +82,31 @@
...
+    end
+
+    it 'converts file_size string in MB into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('0.789 MB')
+      expect(screenshot.file_size_in_bytes).to equal(789000)
+    end
+
+    it 'converts file_size string in GB into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('0.123 GB')
+      expect(screenshot.file_size_in_bytes).to equal(123000000)
```
Comment (cheshire137) [inline spec/rsteamshot/screenshot_spec.rb]: Could add underscores to make it more readable: `123_000_000`.

File: spec/rsteamshot/screenshot_spec.rb [hunk 3]
```diff
@@ -82,4 +82,31 @@
       expect(json['comment_count']).to eq(comment_count)
     end
   end
+
+  describe '#file_size_in_bytes' do
+    it 'converts file_size string in B into integer of bytes' do
+      expect(screenshot).to receive(:file_size).and_return('123 B')
```
Comment (cheshire137) [inline spec/rsteamshot/screenshot_spec.rb]: I usually see `allow` instead of `expect` paired with `to receive`. I guess this works though!

**Review by cheshire137 — APPROVED** [review]
Sorry for the delay reviewing! Thanks for the PR, this looks great! ✨

**Comment by marcqualie** [comment]
Thanks for the feedback. I've updated accordingly.

**Review by cheshire137 — APPROVED** [review]
Thank you!

**Pull request merged**